css-text-4/Overview.bs

```diff
@@ -31,8 +35,519 @@ Introduction</h2>
 <h2 id="transforming">
 Transforming Text</h2>
 
+<h3 id="text-transform-property" oldids="text-transform caps-prop">
+Case Transforms: the 'text-transform' property</h3>
+
 	Issue: Add final level 3 content
 
+<h3 id=word-boundaries>
+Word Boundaries</h3>
+
+	In a number of languages and writing system,
+	such as Japanese or Thai,
+	words are not deliminated by spaces (or any other character)
+	as is the case in English
+	(See <a href="https://www.w3.org/International/articles/typography/linebreak">Approaches to line breaking</a>
+	for a discussion the approach various languages take to word separation and line breaking).
+
+	However, even if text without spaces is the dominant style in such languages,
+	there are cases where making word boundaries (or phrase boundaries) visible
+	through the use of spaces
+	is desired.
+	This is a purely stylistic effect,
+	with no implication on the semantics of the text.
+
+	In Japan for instance, this is commonly done in books for people learning the language--
+	young children or foreign students.
+	People with dyslexia also tend to find this style easier to read.
+
+	The mechanism described in this specification builds upon the existing use
+	of the <{wbr}> element
+	or of U+200B ZERO WIDTH SPACE
+	(See [[UNICODE]])
+	in the document markup as a word (or phrase) delimiter.
+
+	Issue: Should we have a shorthand
+	for the following two properties?
+
+<h4 id=word-boundary-detection>
+Detecting Word Boundaries: the 'word-boundary-detection' property</h4>
+
+	<pre class="propdef">
+	Name: word-boundary-detection
+	Value: normal | manual | auto(<<lang>>)
+	Initial: normal
+	Applies to: [=inline boxes=]
+	Inherited: yes
+	Percentages: N/A
+	Computed value: as specified (However, see special provision for unsupported <<lang>>)
+	Animation type: discrete
+	</pre>
+
+	This property allows the author to decide
+	whether and how
+	the User Agent must analyse the content
+	to determine where word boundaries are,
+	and to insert [=virtual word boundaries=] accordingly.
+
+	A <dfn data-dfn-for='' data-dfn-type=dfn>virtual word boundary</dfn> is similar to the presence
+	of the ZERO WIDTH SPACE (U+200B) character:
+	it introduces a [=soft wrap opportunity=]
+	and is affected by the 'word-boundary-expansion' property;
+	its presence has no effect on text shaping,
+	nor on 'word-spacing'.
+	However, its insertion must have no effect on the underlying content,
+	and must not affect the content of a plain text copy &amp; paste operation.
+
+	<dl dfn-type=value dfn-for=word-boundary-detection>
+		<dt><dfn>manual</dfn>
+		<dd>
+			The User Agent must not insert [=virtual word boundaries=].
```

Comment:
Add some more explanatory text for authors, maybe:

Linguistic analysis is not used to determine [=line wrapping opportunities=] in any language or writing system.

or something better...